lib/standalone.mjs

```diff
@@ -83,6 +84,15 @@ export default async function standalone({
 	let formatterFunction;
 
 	try {
+		/** @see https://github.com/stylelint/stylelint/issues/7447 */
+		if (isString(formatter) && formatter === 'github') {
```

Comment:
question

Is `isString(formatter) &&` necessary?

Reply:
Actually no... even if it is a function or `undefined` the `isString` check is not necessary